test/HealthChecks.Azure.Data.Tables.Tests/AzureTableStorageHealthCheckTests.cs

```diff
@@ -55,36 +56,26 @@ public async Task return_healthy_when_only_checking_healthy_service()
     }
 
     [Fact]
-    public async Task return_healthy_when_checking_healthy_service_and_table()
+    public async Task return_healthy_when_checking_healthy_service_table()
```

Comment:
nit: previously it was checking both service and table, now just table, so I would not use "service" in the name

```suggestion
    public async Task return_healthy_when_checking_healthy_table()
```